akka-typed/src/main/scala/akka/typed/scaladsl/Actor.scala

```diff
@@ -188,6 +188,15 @@ object Actor {
   }
 
   /**
+   * Construct an actor behavior that can react to lifecycle signals only. After spawning this actor
+   * from another actor (or as the guardian of an [[akka.typed.ActorSystem]]) it will be executed
+   * within an [[ActorContext]] that allows access to the system, spawning and watching other
+   * actors, etc.
```

Comment:
Do you think it will always be used together with `deferred`? How would it otherwise get any signals? If that is the case we should perhaps add the `factory: ActorContext[T] ⇒ Behavior[T]` param and automatically wrap it in `deferred`?

Reply:
correction; the param would then be `factory: ActorContext[T] ⇒ Unit`

Reply:
Yeah, makes sense.

Reply:
Done.